pkg/model/core/status_condition.go

```diff
@@ -0,0 +1,5 @@
+package core
+
+import gwv1 "sigs.k8s.io/gateway-api/apis/v1"
+
+const GatewayReasonConflicted gwv1.GatewayConditionReason = "Conflicted"
```

Comment:
should be in controller's package next to GW controller, status update not related to models

Reply:
Good point, will move it there